yolox/layers/jit_ops.py

```diff
@@ -121,21 +121,4 @@ def clear_dynamic_library(self):
 
 
 class FastCOCOEvalOp(JitOp):
```

Comment:
This class should be deleted (same as `COCOeval_opt`) as we do not want to compile any C++ code. 
You should found all usages of this class and make it use regular COCOEval` class and it will not be easy as this class is usually imported dynamically.